geometry_msgs/Vector3 linear
geometry_msgs/Vector3 angular
float32 seconds
string name
string id